config/multirotor/quad/cpd/hitl/wh_config.yaml

```diff
@@ -0,0 +1,1065 @@
+---
```

Comment:
You should be able to remove most of the content of this file and let it inherit the configuration in the config/multirotor/quad/cpd/wh_config.yaml file.  To test it, build the workspace as is, and save it off.  Then delete everything but the "config_base:  ${PROJECT_SOURCE_DIR}" and "commander ->   server_overlay: ${PROJECT_SOURCE_DIR}/config/multirotor/quad/ppd/sitl/server_overlay" keys and rebuild.  Then diff the .yaml files that are generated in both builds and ensure they are identical.  If they are, add your changes and commit.  Its better to reduce the amount of data that we need to maintain, by using the configuration inheritance feature of our build system.

Reply:
You should also diff the .yaml in the parent directory first.  If you have anything different in this yaml, make sure it gets added to the final yaml file.

Reply:
It looks like wh_defgen.py looks for "modules" in wh_config.yaml and fails to generate wh_defs.yaml if "modules" isn't present.